src/openfl/_internal/renderer/opengl/GLRenderer.hx

```diff
@@ -127,9 +125,11 @@ class GLRenderer extends AbstractRenderer {
 	}
 	
 	
-	public function getMatrix (transform:Matrix, snapToPixel: Bool = false):Array<Float> {
+	var getMatrixHelperMatrix = new Matrix ();
+
+	public function getMatrix (transform:Matrix, snapToPixel: Bool = false):Matrix4 {
+		var _matrix = this.getMatrixHelperMatrix;
```

Comment:
this is not part of the shader optimization right ? 
Would separate PR for this make sense ?

Reply:
okay